### PR TITLE
Make logging consistent across control plane and respect log-as-json flag

### PR DESCRIPTION
**Pull request by shubham1172**
# Description

This PR makes logging consistent across various control plane components and makes all log statements respect the `log-as-json` flag. 

After these changes, all the logs are in the correct format and the first two lines are consistent.
image

## Issue reference

Please reference the issue this PR will close: #6693

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [X] Code compiles correctly
* [ ] Created/updated tests
* [ ] Unit tests passing
* [ ] End-to-end tests passing
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Specification has been updated / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Provided sample for the feature / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

File: cmd/injector/main.go [hunk 1]
```diff
@@ -30,26 +30,30 @@ import (
...
-var log = logger.NewLogger("dapr.injector")
+var (
+	log  = logger.NewLogger("dapr.injector")
+	opts = options.New()
+)
 
-func main() {
-	log.Infof("starting Dapr Sidecar Injector -- version %s -- commit %s", buildinfo.Version(), buildinfo.Commit())
+func init() {
+	// Apply options to all loggers
```
Comment (ItalyPaleAle) [inline cmd/injector/main.go:39]: I think this should still go at the beginning of `main` maybe?

Comment (mukundansundar) [reply cmd/injector/main.go:39]: agreed an `init` function is unnecessary here.

File: cmd/injector/main.go [hunk 2]
```diff
@@ -30,26 +30,30 @@ import (
 	"github.com/dapr/kit/logger"
 )
 
-var log = logger.NewLogger("dapr.injector")
+var (
+	log  = logger.NewLogger("dapr.injector")
+	opts = options.New()
```
Comment (mukundansundar) [inline cmd/injector/main.go:35]: options can be declared inside the main function once init is removed.

File: cmd/injector/main.go (hunk 1 above)
Comment (shubham1172) [reply cmd/injector/main.go:39]: Yes, I "reverted" the code to its previous state with `init`, but we can remove it.

**Review by mukundansundar — APPROVED** [review]

**Pull request merged**